README.md

```diff
@@ -2,6 +2,23 @@
 <!-- Edit the title above with your project title -->
 
 ## Project Overview
+### Topic: Can we use ML to make predictions about real estate?
+This topic is important because it can help both businesses and consumers.
```

Comment:
I would elaborate more on the topic and its significance.

Part of validating that this is a real need that requires an IT solution is being able to articulate the impact of that work. What's the size of this industry (consumer, not big sharks real estates). In the current market, why would this be important?